### DISCUSSION TITLE
Can we apply a custom filter before the rotation is applied?

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#7650] [Q&A]
Hi,

Currently a custom filter is applied after the sprite is rotated.
 How can a custom filter be applied before the rotation is applied to the sprite?

**Comment by ivanpopelyshev** [comment]
Filter is applied to rectangular part of screen, because, duh, its a filter. The concept is the same in Flash, SVG, and here. In 3D renderers its named "post-effect".

if you want custom geometry please look at 

https://pixijs.io/examples/#/mesh-and-shaders/triangle-color.js 

and

https://pixijs.io/examples/#/mesh-and-shaders/shadertoy-mesh.js
